credentials/gs1-8110-coupon/credential.json

```
@@ -1,16 +1,17 @@
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://contexts.vcplayground.org/examples/gs1-8110-coupon/v1.json"
+    "https://contexts.vcplayground.org/examples/gs1-8110-coupon/v2.json"
   ],
   "type": ["VerifiableCredential", "GS18110CouponCredential"],
   "issuer": {
     "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
-    "name": "BIG MEGA",
-    "url": "https://example.com",
+    "name": "Local Coffee Shop",
+    "url": "https://example.coffee",
```

I think `example.coffee` is a real registered domain.  Would be better to use an official example domain: https://www.iana.org/domains/reserved.
```suggestion
    "url": "https://coffee.example.com",
```

---

Oh. I thought they had reserved `example.*`...sad.